src/components/phone-input/PhoneInput.tsx

```diff
@@ -123,6 +132,12 @@ export const PhoneInput = ({
   );
 };
 
+function isError(error: string): error is PhoneInputErrorCode {
+  return ['invalid_phone', 'no_associated_account', 'unknown_error'].includes(
```

Comment:
This should refer to constants. Then we would avoid having outdated values, like 'invalid_phone' is here (I think).

Reply:
Updated! Not needed anymore